rxbindings/src/main/java/com/amplifyframework/rx/RxApiBinding.java

```diff
@@ -50,134 +46,44 @@
         this.api = api;
     }
 
-    @NonNull
-    @Override
-    public <T extends Model> Single<GraphQLResponse<Iterable<T>>> query(@NonNull Class<T> modelClass) {
-        return toSingle((onResult, onError) -> api.query(modelClass, onResult, onError));
-    }
-
-    @NonNull
-    @Override
-    public <T extends Model> Single<GraphQLResponse<T>> query(
-            @NonNull Class<T> modelClass, @NonNull String modelId) {
-        return toSingle((onResult, onError) -> api.query(modelClass, modelId, onResult, onError));
-    }
-
-    @NonNull
-    @Override
-    public <T extends Model> Single<GraphQLResponse<Iterable<T>>> query(
-            @NonNull Class<T> modelClass, @NonNull QueryPredicate searchCriteria) {
-        return toSingle((onResult, onError) -> api.query(modelClass, searchCriteria, onResult, onError));
-
-    }
-
     @NonNull
     public <R> Single<GraphQLResponse<R>> query(@NonNull GraphQLRequest<R> graphQlRequest) {
         return toSingle((onResult, onError) -> api.query(graphQlRequest, onResult, onError));
     }
 
-    @NonNull
-    @Override
-    public <T extends Model> Single<GraphQLResponse<Iterable<T>>> query(
-            @NonNull String apiName, @NonNull Class<T> modelClass) {
-        return toSingle((onResult, onError) -> api.query(apiName, modelClass, onResult, onError));
-    }
-
-    @NonNull
-    @Override
-    public <T extends Model> Single<GraphQLResponse<T>> query(
-            @NonNull String apiName, @NonNull Class<T> modelClass, @NonNull String modelId) {
-        return toSingle((onResult, onError) -> api.query(apiName, modelClass, modelId, onResult, onError));
-    }
-
-    @NonNull
-    @Override
-    public <T extends Model> Single<GraphQLResponse<Iterable<T>>> query(
-            @NonNull String apiName, @NonNull Class<T> modelClass, @NonNull QueryPredicate searchCriteria) {
-        return toSingle((onResult, onError) -> api.query(apiName, modelClass, searchCriteria, onResult, onError));
-    }
-
     @NonNull
     @Override
     public <R> Single<GraphQLResponse<R>> query(
             @NonNull String apiName, @NonNull GraphQLRequest<R> graphQlRequest) {
         return toSingle((onResult, onError) -> api.query(apiName, graphQlRequest, onResult, onError));
     }
 
-    @NonNull
-    @Override
-    public <T extends Model> Single<GraphQLResponse<T>> mutate(
-            @NonNull T model, @NonNull MutationType mutationType) {
-        return toSingle((onResult, onError) -> api.mutate(model, mutationType, onResult, onError));
-    }
-
-    @NonNull
-    @Override
-    public <T extends Model> Single<GraphQLResponse<T>> mutate(
-            @NonNull T model, @NonNull QueryPredicate mutationCriteria, @NonNull MutationType mutationType) {
-        return toSingle((onResult, onError) -> api.mutate(model, mutationCriteria, mutationType, onResult, onError));
-    }
-
     @NonNull
     @Override
     public <T> Single<GraphQLResponse<T>> mutate(@NonNull GraphQLRequest<T> graphQlRequest) {
```

Comment:
Not important for this PR, but this should be R, not T.   This was something I missed in my PR.  We can fix it later without it being a breaking change though.